samples/bluetooth/peripheral_uart/README.rst

```diff
@@ -80,11 +80,16 @@ USB CDC ACM extension
 =====================
 
 For the boards with the USB device peripheral, you can build the sample with support for the USB CDC ACM class serial port instead of the physical UART.
-This build uses the UART async adapter module that acts as a bridge between the USB CDC ACM and the default sample configuration.
-The USB CDC ACM provides only the interrupt interface and the default sample configuration uses the UART async API.
+This build uses the custom UART async adapter module that acts as a bridge between the USB CDC ACM and the default sample configuration.
+See :ref:`peripheral_uart_sample_activating_variants` for details about how to build the sample with this extension using the :file:`prj_cdc.conf`.
 
-For more information about the adapter, see the :file:`uart_async_adapter` source files available in the :file:`peripheral_uart/src` directory.
-See :ref:`peripheral_uart_sample_activating_variants` for details about how to build the sample with this extension.
+Async adapter experimental module
+   The default sample configuration uses the UART async API.
+   The UART async adapter creates and initializes an instance of the async module, so that the USB CDC ACM can provide only the interrupt interface.
```

Comment:
The UART async adapter creates and initializes an instance of the async module.
This is needed, because the USB CDC ACM implementation provides only the interrupt interface.